packages/vuetify/package.json

```diff
@@ -80,7 +84,7 @@
     "vue-router": "^3.0.1",
     "vue-template-es2015-compiler": "^1.6.0",
     "vuetify-loader": "^0.2.0",
-    "webpack": "^4.23.1",
+    "webpack": "^4.28.2",
```

Comment:
I prefer to use the same version everywhere if possible, this change (and babel) should be reflected in vuetifyjs.com  too